src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,16 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Machine truck = new Truck();
+        Machine bulldozer = new Bulldozer();
+        Machine excavator = new Excavator();
+
+        Machine[] machines = {truck,bulldozer,excavator};
```

Comment:
There should be a space after the comma in the array initialization to adhere to Java coding standards.